test/literate.litcoffee

```diff
@@ -55,111 +55,3 @@ Tabs work too:
 
 				test "tabbed code", ->
 					ok yes
-
```

Comment:
Perhaps you made a mistake in merging? See in the diff, you deleted a hundred lines of tests from `literate.litcoffee`.